doc/glossary/index.rst

```diff
@@ -91,13 +91,18 @@ Glossary
 
 
   Qt
-      `Qt <http://trolltech.com/products/qt/>`__ is a cross-platform
+      `Qt <http://www.qt.io/>`__ is a cross-platform
       application framework for desktop and embedded development.
 
   Qt4
-      `Qt4 <http://trolltech.com/products/qt/>`__ is the most recent
-      version of Qt cross-platform application framework for desktop
-      and embedded development.
+      `Qt4 <http://doc.qt.io/qt-4.8/index.html>`__ is the previous,
+      but most widely used, version of Qt cross-platform application
+      framework for desktop and embedded development.
+
+  Qt5
+      `Qt5 <http://doc.qt.io/qt-5/index.html>`__ is the previous,
```

Comment:
Gah, good catch.  Fixed and force-pushed.

On Fri, Aug 14, 2015 at 11:40 AM Jens Hedegaard Nielsen <
notifications@github.com> wrote:

> In doc/glossary/index.rst
> <https://github.com/matplotlib/matplotlib/pull/4922#discussion_r37088789>:
>
> >        application framework for desktop and embedded development.
> >
> >    Qt4
> > -      `Qt4 <http://trolltech.com/products/qt/>`__ is the most recent
> > -      version of Qt cross-platform application framework for desktop
> > -      and embedded development.
> > +      `Qt4 <http://doc.qt.io/qt-4.8/index.html>`__ is the previous,
> > +      but most widely used, version of Qt cross-platform application
> > +      framework for desktop and embedded development.
> > +
> > +  Qt5
> > +      `Qt5 <http://doc.qt.io/qt-5/index.html>`__ is the previous,
>
> Looks like copy paste ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/4922/files#r37088789>.
>